Assets/Scripts/Gameplay/Interactor.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class Interactor : MonoBehaviour
```

Comment:
To me, this file name is quite unclear as to what it does. Could you give it a more descriptive name?